### ISSUE TITLE
Link icon has no alt text

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4868]
The link icon mostly found on the Creators page (a few other occurrences) doesn’t seem to have an alt text. Consider either giving it an alt text, or marking it as decorative if it is just for presentation purposes.
